org-gtd.el

```diff
@@ -148,7 +150,7 @@
   "How to identify projects in the GTD system.")
 
 (defconst org-gtd-stuck-projects
-  '("+LEVEL=2-DONE+CATEGORY=\"Projects\"" ("TODO" "NEXT" "WAIT") nil "")
+  '("+LEVEL=2-TODO=\"DONE\"+CATEGORY=\"Projects\"" ("TODO" "NEXT" "WAIT") nil "")
```

Comment:
So obviously I don't have tests for this package right now - did you test that this implementation works?

Reply:
I've not created a test if that is what you mean. I'm still very much learning things. That said, this change simply explicitly follows the documentation.